ams/server/src/main/java/com/netease/arctic/server/dashboard/utils/OptimizingUtil.java

```diff
@@ -63,34 +61,34 @@ private static FilesStatistics collectOptimizingFileInfo(MetricsSummary metricsS
     }
     return FilesStatistics.builder()
```

Comment:
I notice that `reRowDeletedDataFileCnt` is missing here. It may be a bug, and we can fixed it with another PR.

Reply:
Yes, it is missing here.